src/d4bl/services/langfuse/parsers.py

```
@@ -5,6 +5,16 @@
 from typing import Any, Dict
 
 
+def keyword_relevance(query: str, text: str) -> float:
+    """Score relevance of *text* to *query* via simple keyword overlap (1.0–5.0)."""
```

Ruff (0.15.2)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)